DNN Platform/Library/Security/PortalSecurity.cs

```diff
@@ -280,20 +328,45 @@ public static bool IsInRoles(UserInfo objUserInfo, PortalSettings settings, stri
             return isInRoles;
         }
 
+        /// <summary>
+        /// Determines whether the specified user is a friend of the current user.
+
+        /// </summary>
+        /// <param name="userId">The user identifier.</param>
+        /// <returns>
+        ///   <c>true</c> if the specified user is a friend of the current user; otherwise, <c>false</c>.
+
+        /// </returns>
         public static bool IsFriend(int userId)
         {
             UserInfo objUserInfo = UserController.Instance.GetCurrentUserInfo();
             PortalSettings settings = PortalController.Instance.GetCurrentPortalSettings();
             return IsInRoles(objUserInfo, settings, RoleFriendPrefix + userId);
         }
 
+        /// <summary>
+        /// Determines whether the specified user is a follower of the current user.
+
+        /// </summary>
+        /// <param name="userId">The user identifier.</param>
+        /// <returns>
+        ///   <c>true</c> if the specified user is a follower of the current user; otherwise, <c>false</c>.
+
+        /// </returns>

```

Comment:
```suggestion
        /// <summary>
        /// Determines whether the specified user is a follower of the current user.
        /// </summary>
        /// <param name="userId">The user identifier.</param>
        /// <returns>
        ///   <c>true</c> if the specified user is a follower of the current user; otherwise, <c>false</c>.
        /// </returns>
```